config/utils/unescapeHtmlString.ts

```diff
@@ -0,0 +1,11 @@
+import { NullableString } from "../types/root_types";
+
+export function unescapeHtmlString(htmlStr: NullableString): string {
```

Comment:
Can we use a function to replace all HTML entities instead of just these? (i.e. semicolon, colon, hyphen, ..)

I think this solution would work. We should put it behind a try/catch though to avoid the feed breaking though.
https://stackoverflow.com/a/7394814/1869326

Reply:
This only parses numerics. I'll add on top of this. The solution is O(n). Should be fast anyway